imagick.c

```diff
@@ -1051,52 +1060,76 @@ PHP_MINIT_FUNCTION(imagick)
 #endif
 
 	php_imagick_sc_entry = zend_register_internal_class(&ce TSRMLS_CC);
+#if PHP_VERSION_ID >= 80300
+	php_imagick_sc_entry->create_object = php_imagick_object_new;
```

Comment:
create object affectation can be done this way at least since 8.1, just wanted to reduce complexity and avoid additional conditions.